### PR TITLE
Sandrine's weekly covid-19 evaluation

### PR DESCRIPTION
**Pull request by MindstormingAB**
Link to my deployed project: https://weekly-evaluation.netlify.app/

My reflections on how this week's project turned out: I didn't have so much time to spend on this project so I mainly focused on the blue requirements.

Things I'd like to have clarified or explained in more detail: I was not able to add a footer as a React component that will behave in a correct way for all screen sizes.

### CODE REVIEW

File: code/README.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+The first question is displayed when the Start button is clicked.
+The following questions are displayed when the previous one is answered.
+When the last question is answered, the button Submit is revealed.
+By clicking on Submit, a summary is displayed instead of the survey form.
 
-1. On GitHub, click the 'use this template' button to create a new repo on your account
-1. Clone the new repo to your computer
-1. Install required dependencies with npm: `npm install`
-1. Start the project: `npm start`
+The survey form is responsive and works in different browsers (Chrome, Firefox, Edge and Safari) and with ChromeVox Classic Extension.
```
Comment (sandra-shumanteva-kaevska) [inline code/README.md:13]: Great job. Even that wasn't requirement you did it.

File: code/src/Restart.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+const Restart = () => {
+  const resetSurvey = () => {
+    window.location.reload();
+  }
```
Comment (sandra-shumanteva-kaevska) [inline code/src/Restart.js:7]: You don't need to reload the page, by resetting the state variables into their initial values, the form will go to it's initial state. For example:
`setShowQuestion(false);
  setPlace('');
 setRating('');
  setFeedback('');
 setShowSummary(false);`

File: code/src/Submit.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Submit = ({ onClick }) => {
```
Comment (sandra-shumanteva-kaevska) [inline code/src/Submit.js:3]: This Submit component is the same like Start component. You can combine both of them into one button component, that will also have a title like parameter

**Review by sandra-shumanteva-kaevska — COMMENTED** [review]
You have very god job done. I think you understand components, props and state. But I also think that form logic should be simplified and some components can be reused, like button component.

File: code/src/Submit.js (hunk 3 above)
Comment (MindstormingAB) [reply code/src/Submit.js]: Thank you so much for this comment Sandra! I replaced my 3 buttons with one unique button 🥳

File: code/src/Form.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      ) : (!showSummary && !showQuestion) ? (
+        <form>
+          <Intro />
+          <Start onClick={handleStart} />
+        </form>
+      ) : (
+                  <Summary place={place} rating={rating} feedback={feedback} />
+                )}
+    </section>
+  )
```
[the remark this answers is not in the document]
Comment (MindstormingAB) [reply code/src/Form.js:53]: OMG! Great! I struggled a while with these conditions and your proposition is brilliant! 🙏

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Changing to CR because of inconsistent use of semicolons, you should either have them everywhere or not at all :)

**Comment by MindstormingAB** [comment]
@HIPPIEKICK Added semicolons everywhere (I hope 🤔)

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**